kong/plugins/acme/client.lua

```diff
@@ -13,6 +13,84 @@ local CERTKEY_KEY_PREFIX = "kong_acme:cert_key:"
 
 local LOCK_TIMEOUT = 30 -- in seconds
 
+local function get_allowed_config_domains(host)
```

Comment:
moved `get_allowed_config_domains` , `load_domain` &  `is_domain_db_config_exists` to `client.lua` to prevent circular imports 

i.e. 
handler.lua -> client.lua -> handler.lua